packages/devextreme-scss/scss/widgets/generic/chat/_sizes.scss

```diff
@@ -51,12 +64,16 @@ $chat-messagelist-day-header-first-padding-top: 20px !default;
   $chat-errors-row-gap: 4px !default;
   $chat-error-padding-block: 9px !default;
 
-  $chat-messagelist-empty-icon-box-size: 64px !default;;
+  $chat-messagelist-empty-icon-box-size: 64px !default;
   $chat-messagelist-empty-icon-size: 40px !default;
   $chat-messagelist-empty-icon-margin-bottom: 12px !default;
   $chat-messagelist-empty-row-gap: 4px !default;
   $chat-messagelist-empty-message-font-size: 18px !default;
   $chat-messagelist-empty-prompt-font-size: 14px !default;
+
+  $chat-typingindicator-bubble-height: 2.571429em !default;
+  $chat-typingindicator-bubble-width: 5.428571em !default;
```

Comment:
It should be 24px instead of 28px according to the design

Reply:
This is not a true statement because the size of the bubble that contains the text is 28px. I believe this is a design error.